loadtest/user/userentity/websocket.go

```diff
@@ -127,7 +127,8 @@ func (ue *UserEntity) listen(errChan chan error) {
 		}
 
 		client.Listen()
-		chanClosed := false
+		var chanClosed bool
+		var timedOut bool
```

Comment:
This is pretty explicit. We could replace it with a single `var exitLoop bool` if you think the double variable is a bit excessive.

Reply:
Seems fine to me.